src/threepp/renderers/gl/GLCapabilities.hpp

```diff
@@ -59,7 +59,7 @@ namespace threepp::gl {
 
     private:
         GLCapabilities()
-            : maxAnisotropy(glGetParameter(GL_MAX_TEXTURE_MAX_ANISOTROPY)),
+            : maxAnisotropy(0),
```

Comment:
this one is not supported for gl 4.1

Reply:
This flag has very limited usage in the [three.js codebase](https://github.com/mrdoob/three.js/search?q=maxAnisotropy), so we can probably remove it altogether